src/features/basic/slider-improvements/slider-improvements.ts

```diff
@@ -0,0 +1,130 @@
+import $style from './slider-improvements.module.css';
+import SliderButton from '@src/features/basic/slider-improvements/SliderButton.vue';
+import { clickElement } from '@src/util';
+
+function onTileReady(tile: PrunTile) {
+  sliderMinMaxButtons(tile);
+  sliderTableBulkMinMax(tile);
+}
+
+function sliderTableBulkMinMax(tile: PrunTile) {
+  subscribe($$(tile.anchor, 'table'), table => {
+    const headerRow = _$(table, 'thead')?.firstElementChild;
+    if (!headerRow) return;
+    const sliders = _$$(table, 'rc-slider');
+    if (sliders.length > 0) {
```

Comment:
This `if` could be inverted to `sliders.length === 0` to reduce code nesting with an early `return`